src/composables/draw/draw-notifications.composable.ts

```diff
@@ -0,0 +1,47 @@
+import { watch } from 'vue'
+import { storeToRefs } from 'pinia'
+import { useTranslation } from 'i18next-vue'
+
+import { useAlertNotificationsStore } from '@/stores/alert-notifications.store'
+import { useDrawStore } from '@/stores/draw.store'
+import { DrawStateActive } from '@/stores/draw.store.model'
+
+export default function useDrawNotifications() {
+  const { t } = useTranslation()
+  const { addNotification } = useAlertNotificationsStore()
+  const { drawStateActive } = storeToRefs(useDrawStore())
+
+  watch(drawStateActive, notify)
+
+  function notify(drawStateActive: DrawStateActive) {
+    let notification: string | undefined
+
+    switch (drawStateActive) {
+      case 'drawPoint':
+        notification = t('Click to draw the point')
+        break
+      case 'drawLine':
+        notification = t(
+          'Click to start drawing line<br>Double-click to finish'
+        )
+        break
+      case 'drawPolygon':
+        notification = t(
+          'Click to start drawing polygon<br>Double-click or click last point to finish'
+        )
+        break
+      case 'drawCircle':
+        notification = t('Click to start drawing circle')
+        break
+      case 'drawLabel':
+        notification = t('Click to place the label')
+        break
+    }
+
+    if (notification) {
```

Comment:
maybe add en explicit case for
if (drawStateActive === undefined {
return {}
}

then you don't need this check

and even better it would be nice to refacter the undefined value of the type DrawStateActive to something explicit, e.g. 'drawInactive'

(nice to have)

Reply:
```
maybe add en explicit case for
if (drawStateActive === undefined {
return {}
}

then you don't need this check
```
Ok will do.

`and even better it would be nice to refacter the undefined value of the type DrawStateActive to something explicit, e.g. 'drawInactive'`
You are right, but maybe wait for the stability of https://camptocamp.atlassian.net/browse/GSLUX-704 before making some refactoring, won't do 
 in this PR as it concerns notification messages only.